docs/design/job_architecture.md

```diff
@@ -278,128 +172,67 @@ These are all caught by the `JobCommChannel` on the browser side, then parsed an
 }
 ```
 
-For example:
-```json
-{
-  "msg_id": "some_string",
-  "data": {
-    "msg_type": "job_status",
-    "content": {
-      "example_job_id": {
-        "jobState": {
-          "job_id": "example_job_id",
-          "status": "running",
-          ... other state keys ...
-        },
-        "outputWidgetInfo": {}
-      }
-    }
-  }
-}
-```
-
 These are described below. The name (`msg_type`) is given, followed by the keys given in the `content` block.
 
-By design, these should only be seen by the `JobCommChannel` instance, then sent into bus messages that get sent on specific channels. That information is also given in each block.
+The backend bundles together multiple messages of the same type in an object indexed by key (usually job or cell ID). In nearly all cases, the frontend then separates out the data and sends it out on individual channels for each job, batch job, or cell.
 
-The backend bundles together multiple messages of the same type in an object indexed by key (usually job or cell ID). In nearly all cases, the frontend then separates out the data and sends it out on individual channels for each job or cell.
+By design, these messages are only seen by the `JobCommChannel`; the JobComm interface does not constitute an API for narrative job data.
 
 ### `ERROR`
-A general job comm error, capturing most errors that get thrown by the kernel
 
-**content** (this varies, but usually includes the below)
-  * `raw_request` - the original request message that wound up in an error
-  * `source` - request type or method that triggered the error
+A general job comm error message, capturing exceptions that get thrown by the kernel. The frontend uses the `request` data to distribute error messages on the appropriate job or cell channels.

```

Comment:
The message types use the JCM values, but there's such a variety of different data structures that come from the backend that it would not be useful or practical to use the controlled vocabulary.

Reply:
Can you add that to the docs?

Reply:
:+1: